.github/workflows/build_packages.yml

```diff
@@ -134,3 +136,18 @@ jobs:
         allowUpdates: true
         replacesArtifacts: true
         makeLatest: false
+
+  test:
+    name: "Trigger test packages"
+    needs: build_packages
+    runs-on: ubuntu-24.04
+    steps:
+      - name: "Invoke workflow :: Test Packages"
+        uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc # v1.2.4
+        with:
+          workflow: Validate and Publish Release
```

Comment:
Does this "Validate and Publish Release" workflow exist? Did you meant to invoke "Test packages" instead?

Reply:
Must be `Test packages` -.-